src/components/pages/VolunteersPage/VolunteersPage.js

```diff
@@ -4,15 +4,21 @@ import {
   Container,
   FormControl,
   InputGroup,
-  CardColumns
+  Row,
+  Col,
+  CardColumns,
+  Tooltip,
+  OverlayTrigger
 } from 'react-bootstrap';
 import VolunteerCard from '../../VolunteerCard';
 import volunteerActions from '../../../_actions/volunteer.actions';
 import Layout from '../../Layout/Layout';
+import InviteVolunteerModal from './InviteVolunteerModal/InviteVolunteerModal';
```

Comment:
nit: The import path can just be `./InviteVolunteerModal` since it is exported from `index.js`.

Reply:
OOooooh hello there